querybook/webapp/components/DataTableViewColumn/DataTableViewColumn.tsx

```diff
@@ -21,20 +21,44 @@ export interface IDataTableViewColumnProps {
     onEditColumnDescriptionRedirect?: Nullable<() => Promise<void>>;
 }
 
+type ColumnOrderBy = 'alphabetical' | 'usage';
+
+const COLUMN_STATS_USAGE_KEY = 'usage';
+
 export const DataTableViewColumn: React.FunctionComponent<
     IDataTableViewColumnProps
 > = ({
     updateDataColumnDescription,
     table = null,
-    tableColumns = [],
     numberOfRows = null,
     onEditColumnDescriptionRedirect,
 }) => {
     const [filterString, setFilterString] = React.useState('');
-    const [orderColumnsBy, setOrdeColumnsBy] = React.useState(true);
-    const [orderBoardBy, setOrderBoardBy] = React.useState(false);
+    const [orderColumnsBy, setOrderColumnsBy] =
+        React.useState<ColumnOrderBy>('usage');
```

Comment:
COLUMN_STATS_USAGE_KEY?

Reply:
if there is no usage, what would happen?

Reply:
This 'usage' string just corresponds to the column order by type (so shouldn't use the constant here). If there's no usage, they're all treated as 0 usage. But I updated it so that you can't see the sort by usage option if there are no usages available for any of the columns so it isn't confusing to users.